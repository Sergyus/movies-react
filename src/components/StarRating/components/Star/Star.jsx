import React from 'react';
import styles from './Star.scss';

const Star = () => (
  <div
    className={styles.star}
  />
);


export default Star;
