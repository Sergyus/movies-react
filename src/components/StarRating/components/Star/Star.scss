@import "../../../../styles/variables.scss";

.star {
  height: 20px;
  width: 20px;
  margin: 2px;
  float: left;
  background-color: $blue-color;
  clip-path:
    polygon(
      50% 0%,
      63% 38%,
      100% 38%,
      69% 59%,
      82% 100%,
      50% 75%,
      18% 100%,
      31% 59%,
      0% 38%,
      37% 38%
    );
}
