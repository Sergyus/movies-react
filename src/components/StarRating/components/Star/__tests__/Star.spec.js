import React from 'react';
import { create } from 'react-test-renderer';
import Star from '..';

describe('Star component renders correctly', () => {
  it('Selected Star renders correctly', () => {
    const tree = create(<Star isSelected />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
