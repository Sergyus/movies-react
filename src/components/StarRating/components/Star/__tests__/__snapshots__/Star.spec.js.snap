// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Star component renders correctly Selected Star renders correctly 1`] = `
<div
  className="star"
/>
`;
